azure-pipelines-azurezagent/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:20.04
+RUN DEBIAN_FRONTEND=noninteractive apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get upgrade -y
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends \
+    apt-transport-https \
+    apt-utils \
+    ca-certificates \
+    curl \
+    git \
+    iputils-ping \
+    jq \
+    lsb-release \
+    software-properties-common \
+    libicu66
+
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
+
+# Can be 'linux-x64', 'linux-arm64', 'linux-arm', 'rhel.6-x64'.
+ENV TARGETARCH=linux-x64
+ENV RELEASE=3.226.3
```

Comment:
ideally we can let the github action pipeline call to github to figure out the latest version, use it and tag the container with it. Can we inject an ENV to the build like that?

```
curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/microsoft/azure-pipelines-agent/releases/latest | jq .name
```

Reply:
You can inject arguments like this:
`docker build -t [image-name] --build-arg [arg-variable]=[value] .`

Reply:
fixed